### PR TITLE
Prepare liana v7.0 release

### PR DESCRIPTION
**Pull request by edouardparis**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
 # Liana daemon and GUI release notes
 
+## 7.0
+
+This release introduces support for Wizardsardine servers as an optional backend for the Liana GUI
```
Comment (darosior) [inline CHANGELOG.md:5]: Do we call it Wizardsardine servers now? I thought it was Liana Connect? (also present below)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+
+
+#### Liana GUI
+
+- The user does not have anymore to select a network at start up, the launcher looks for a installed
+  network in the following order: `mainnet`, `testnet`, `signet`, `regtest`.
+- The launcher reads the `settings.json` file and provide information about the installed wallet or
+  provide the user a choice to create or add a wallet.
+- When creating or adding a wallet for the selected network, user has the choice to use his local
+  node or use the Wizardsardine servers has a backend for the Liana GUI.
```
Comment (jp1ac4) [inline CHANGELOG.md:30]: "as a backend"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,59 @@
...
+## 7.0
+
+This release introduces support for Wizardsardine servers as an optional backend for the Liana GUI
+and Electrum server as an optional node interface for the Liana daemon.
+
+### Breaking changes
+
+Liana Daemon has a migration for its database to add a new table `transactions` to keep in cache
+transactions to reduce requests to the node. Once Liana GUI or `lianad` run for the first time
+this migration is applied and cannot be reverted.
```
Comment (darosior) [inline CHANGELOG.md:12]: A bit too many technical details IMO, we should just let the reader now what it means for them.

```suggestion
Running Liana v7 on an existing installation will migrate its database. Once migrated the database won't be compatible with previous versions of Liana. This means you won't be able to open with Liana v6 (or below) any wallet opened or created with Liana v7.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,59 @@
...
+  wrapper.
+- We add the minimum feerate requested in the error message of the create spend command.
+
+#### Liana GUI
+
+- The rendering of amounts superior to thousands includes now space.
+- We detect if the descriptor changed before displaying registration status in the installer.
+- We fixed some display issues with overlapping text, notably in the transaction panel.
+- On reload, new labels are correctly loaded in existing transaction list in the home panel.
+- In order to handle flaky connection, the whole harware wallet list is reset at each step
```
Comment (jp1ac4) [inline CHANGELOG.md:54]: "hardware"

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,59 @@
...
+
+Liana Daemon has a migration for its database to add a new table `transactions` to keep in cache
+transactions to reduce requests to the node. Once Liana GUI or `lianad` run for the first time
+this migration is applied and cannot be reverted.
+
+### Features
+
+#### Liana daemon / library
+
+- We added a anti fee snipping protection to the create spend transaction command.
```
Comment (darosior) [inline CHANGELOG.md:18]: nit, feel free to ignore
```suggestion
- Transactions are now created with fee sniping protection.
```

**Review by jp1ac4 — COMMENTED** [review]
A couple of small changes.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,59 @@
...
+- We added optional support for an Electrum server as interface to the bitcoin blockchain instead
+  of Bitcoin Core.
```
Comment (darosior) [inline CHANGELOG.md:20]:
```suggestion
- You can now remotely connect to your node using Electrum, as an alternative to running Bitcoin Core locally.
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,59 @@
...
+
+
+#### Liana GUI
+
+- The user does not have anymore to select a network at start up, the launcher looks for a installed
+  network in the following order: `mainnet`, `testnet`, `signet`, `regtest`.
```
Comment (darosior) [inline CHANGELOG.md:26]:
```suggestion
- The user does not have to select a network at start up anymore. The launcher looks for an installed
  network in the following order: `mainnet`, `testnet`, `signet`, `regtest`.
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,59 @@
...
+- The launcher reads the `settings.json` file and provide information about the installed wallet or
+  provide the user a choice to create or add a wallet.
```
Comment (darosior) [inline CHANGELOG.md:28]:
```suggestion
- The launcher reads the `settings.json` file and provides information about the installed wallet or
  provides the user with a choice to create or add a wallet.
```

File: CHANGELOG.md (hunk 2 above)
Comment (darosior) [inline CHANGELOG.md:30]: Liana connect vs Ws servers again. Also, the user can use a remote Electrum server so the alternative isn't only his local node. Finally: s/has/as/.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,59 @@
...
+- When creating or adding a wallet for the selected network, user has the choice to use his local
+  node or use the Wizardsardine servers has a backend for the Liana GUI.
+- User can share his hardware wallets xpubs by clicking on "Share xpubs" from the launcher.
+- Wallet policy description was added to the installer and in the wallet settings.
+- If using the Wizardsardine servers as remote backend, a user can provide access to a wallet to
+  an other user by sending him an invitation with the invitee email in the wallet settings.
+- Coldcard and Specter DIY can now be used for any taproot setup.
```
Comment (darosior) [inline CHANGELOG.md:35]:
```suggestion
- Coldcard and Specter DIY can now be used with taproot descriptors.
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,59 @@
...
+- The software version is now in the title bar.
```
Comment (darosior) [inline CHANGELOG.md:36]: It sounds like it was moved.

```suggestion
- We now display the software version in the title bar.
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,59 @@
...
+- A `--help` flag was added to display a help menu about the available flags.
+- Bitcoin Core 27.1 is now downloaded if user chooses Liana managed node during install.
```
Comment (darosior) [inline CHANGELOG.md:38]:
```suggestion
- The Bitcoin Core node version to install for managed local nodes was upgraded to 27.1.  
```

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,5 +1,59 @@
...
+
+### Fixes
+
+#### Liana daemon / library
+
+- We check that the change multipath is not hardened while importing descriptor in LianaDescriptor
+  wrapper.
```
Comment (darosior) [inline CHANGELOG.md:45]:
```suggestion
- We now check the change multipath is not hardened when importing a descriptor (or creating a `LianaDescriptor` struct).
```

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,5 +1,59 @@
...
+- We add the minimum feerate requested in the error message of the create spend command.
```
Comment (darosior) [inline CHANGELOG.md:46]: I'm not sure to understand, did we add the requested feerate, or the minimum feerate?

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,5 +1,59 @@
...
+
+#### Liana GUI
+
+- The rendering of amounts superior to thousands includes now space.
```
Comment (darosior) [inline CHANGELOG.md:50]:
```suggestion
- The rendering of amounts superior to thousands now includes space.
```

**Review by darosior — COMMENTED** [review]
A bunch of change requests for the CHANGELOG. Sorry to be nit-picky but i think the changelog is an important communication medium.

**Comment by darosior** [comment]
(I only commented about what was present and haven't been through the git history to check nothing was missing.)

File: CHANGELOG.md (hunk 13 above)
Comment (edouardparis) [reply CHANGELOG.md:46]: the minimum https://github.com/wizardsardine/liana/pull/1186

**Review by edouardparis — COMMENTED** [review]
Self-ACK d95e792f56a9b9bcc3f8dd0c8fea43debbff36be

**Pull request merged**